httping.go

```diff
@@ -148,8 +150,17 @@ func ping(httpVerb string, url *url.URL, count int, hostHeader string, jsonResul
 
 	// Change request timeout to 2 seconds
 	timeout := time.Duration(2 * time.Second)
+	
+	// set up proxy (if any)
+	transport := &http.Transport{}
+
+	if !noProxy {
+		transport.Proxy = http.ProxyFromEnvironment
```

Comment:
Is it possible to print if we are using a proxy and which one? it would be helpful when using this tool for debugging. Something like "Using proxy: XYZ" / "Not using proxy"

Reply:
Sure! TODO

Reply:
Addressed in https://github.com/pjperez/httping/pull/15/commits/e061ee2dde758a7baca0380775fdcbec4a0d11c4 (and previous commits). I used a snippet from here: https://github.com/keyring-so/keyring-desktop/blob/9c6ca18257fee150f922d7559a85e7270373bcdc/app.go#L80

Things to note:
* Instead of "Using proxy: XYZ" / "Not using proxy", I made the program say either "proxy=None" or "proxy={{proxy}}". This is to save space, and to match program's vibe (everything's lowercase, `=` in output)
* Proxy information is placed after the URL and before seq. I think it's a good placement, but feel free to play with the string and place it elsewhere.
* When proxy is set, the program says: "proxy=**WinHTTP:NamedProxy|://**{{ipAddress}}:{{port}}" . This is the same behavior as here: https://github.com/keyring-so/keyring-desktop/blob/9c6ca18257fee150f922d7559a85e7270373bcdc/app.go#L80 . *I guess* it's possible to remove the part I highlighted (at worst, through string operations), but I'm not sure if it's needed
* Proxy information is also displayed for ping timeouts. I think it is needed there. If it's not needed there , I can remove it

Let me know what you think. Thanks!

Reply:
Sounds good to me, thank you for adding this feature 🙏